tools/boards.txt.py

```diff
@@ -889,6 +890,17 @@
         ( '.menu.UploadSpeed.921600.upload.speed', '921600' ),
         ]),
 
+    ####################### flash erase
+
+    'flash_erase_menu': collections.OrderedDict([
+        ( '.menu.FlashErase.none', 'Only Sketch' ),
+        ( '.menu.FlashErase.none.upload.erase_cmd', '' ),
+        ( '.menu.FlashErase.sdk', 'SDK Settings' ),
```

Comment:
I suggest "Sketch + SDK Settings" here, to reflect reality.

Reply:
+1 (or 'Sketch + WiFi settings' ?)